ibis/backends/flink/__init__.py

```diff
@@ -412,7 +446,19 @@ def create_view(
         Table
             The view that was created.
         """
-        raise NotImplementedError
+        if obj is None:
+            raise exc.IbisError("The obj parameter is required")
+
```

Comment:
Do we want to make this a required positional argument instead and skip the `None` handling?